ch_backup/backup/metadata/backup_metadata.py

```diff
@@ -178,6 +191,7 @@ def load(cls, data: dict) -> "BackupMetadata":
             backup.size = meta["bytes"]
             backup.real_size = meta["real_bytes"]
             backup._state = BackupState(meta["state"])
+            backup._fail_reason = meta["fail_reason"] if "fail_reason" in meta else None
```

Comment:
Suggest to replace `meta["fail_reason"] if "fail_reason" in meta else None` with `meta.get("fail_reason", None)` for simplicity and consistency with the code below.